app/views/layouts/_footer.html.erb

```diff
@@ -19,10 +19,10 @@
 
         <div class="col-xs-3">
           <h4>SOCIAL MEDIA</h4>
-          <p><%= image_tag("facebook.jpg") %> Facebook</p>
-          <p><%= image_tag("twitter.jpg") %> Twitter</p>
-          <p><%= image_tag("instagram.jpg") %> Instagram</p>
-          <p><%= image_tag("pinterest.jpg") %> Pinterest</p>
+          <p><%= image_tag("facebook.jpg") %> <a href="https://www.facebook.com/sharer/sharer.php?u=http://colorfulblind.herokuapp.com/">Facebook</a></p>
```

Comment:
Could you try to use [Font Awesome](http://fortawesome.github.io/Font-Awesome/icons/#brand) social media icons?